android/src/main/java/com/twiliovoicereactnative/AudioManager.java

```diff
@@ -0,0 +1,157 @@
+package com.twiliovoicereactnative;
+
+import android.content.Context;
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+
+import com.twilio.audioswitch.AudioDevice;
+import com.twilio.audioswitch.AudioSwitch;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * AudioManager singleton class. Maintains a persistent AudioSwitch object and listens for audio
+ * device changes. Generates UUIDs per audio device when the AudioSwitch library updates available
+ * audio devices.
+ */
+@RequiresApi(api = Build.VERSION_CODES.N)
+public class AudioManager {
```

Comment:
Can you choose a different name to not get confused with Android AudioManager? https://developer.android.com/reference/android/media/AudioManager